biogrid/libs/biogrid-simulator/src/lib/bioenergy-source/solar-panel.ts

```diff
@@ -0,0 +1,53 @@
+import { getSunlight } from '@biogrid/weather';
+import {
+  EnergySource,
+  Validatable,
+  validate,
+  Power,
+  SunlightIntensity,
+} from '@biogrid/grid-simulator';
+
+export class SolarPanel implements EnergySource {
+  // Area in meters squared
+  private area: number;
+  private efficiency: number;
```

Comment:
can we also have a comment on what the efficiency measure looks like? is it a percent efficiency? if so, is it on a scale of 0-1, or 0-100?